tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -305,7 +306,7 @@ public function testImportDetectsNotImportedAnnotation()
     public function testImportDetectsNonExistentAnnotation()
     {
         $reader = $this->getReader();
-        $reader->getPropertyAnnotations(new \ReflectionProperty(TestNonExistentAnnotationClass::class, 'field'));
+        return $reader->getPropertyAnnotations(new \ReflectionProperty(TestNonExistentAnnotationClass::class, 'field'));
```

Comment:
why adding such return statements in tests which are asserting the exception ?

Reply:
The idea was to use assert in child class then. Currently child class does not run any assertions. What do you suggest @stof ?

Reply:
Took me a while to figure out how this worked. Instead of adding a `return` statement here, I'd add a `@doesNotPerformAssertions` annotation on the risky tests in `SimpleAnnotationReaderTest`: these tests check for the absence of an exception without asserting anything. Simply asserting for a return type for the sake of making an assertion is useless IMO, we might as well tell PHPUnit that we're not performing an assertion.

As an alternative, you can add appropriate assertions for the tests, but I'd skip that.